### PR TITLE
Add support to negative axis value for squeeze(x, axis) of theano backend.

### PR DESCRIPTION
**Pull request by w1d2s**
Add support to negative axis value for squeeze(x, axis) of theano backend.
```python
import keras.backend as K
import numpy as np
a = np.random.uniform(low=-1, high=1, size=(3, 1, 1))
b = K.variable(a, name='b')
print b.eval()
"""
[[[-0.38140643]]

 [[-0.81984013]]

 [[ 0.21176814]]]
"""
bb = K.squeeze(b, axis = -1)
print bb.eval()
"""
[[-0.38140643]
 [-0.81984013]
 [ 0.21176814]]
"""
```

### CODE REVIEW

File: keras/backend/theano_backend.py [hunk 1]
```diff
@@ -398,6 +398,8 @@ def expand_dims(x, dim=-1):
 def squeeze(x, axis):
     '''Remove a 1-dimension from the tensor at index "axis".
     '''
+    if axis < 0
+        axis = x.ndim + axis
```
Comment (fchollet) [inline keras/backend/theano_backend.py]: If think you mean `axis % ndim`. Is this consistent with the TF backend API?
